java/java/JavaParser.g4

```diff
@@ -39,7 +39,7 @@ options {
 }
 
 compilationUnit
-    : packageDeclaration? (importDeclaration | ';')* (typeDeclaration | ';')*
+    : packageDeclaration? (importDeclaration | ';')* (typeDeclaration | ';')* EOF
```

Comment:
Good catch. Malformed start rules, where EOF is not on all ats, and more generally, an ATN that does not exit with EOF, will need to be detected. https://github.com/kaby76/g4-scripts/issues/5